lib/api/kuzzle.js

```diff
@@ -42,18 +43,18 @@ const
   TokenManager = require('./core/auth/tokenManager'),
   Validation = require('./core/validation'),
   InternalIndexBootstrap = require('./core/storage/bootstrap/internalIndexBootstrap'),
-  IndexCache = require('./core/storage/indexCache'),
-  IndexEngine = require('./core/storage/indexEngine'),
+  IndexStorage = require('./core/storage/indexStorage'),
+  CacheEngine = require('./core/cache/cacheEngine'),
+  StorageEngine = require('./core/storage/storageEngine'),
   Vault = require('./core/vault'),
   runShutdown = require('../util/shutdown'),
-  Services = require('../services'),
   Logger = require('../util/log');
 
 // Add meaningful stacktrace in Node.js 10
 // @todo must be removed in Node.js 12
-if (process.env.NODE_ENV === 'development') {
-  require('../util/trace');
-}
+// if (process.env.NODE_ENV === 'development') {
+//   require('../util/trace');
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this commented out code. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.kaliop.net/coding_rules#rule_key=javascript%3ACommentedCode)